gdx/src/com/badlogic/gdx/graphics/g2d/Sprite.java

```diff
@@ -37,6 +37,7 @@ public class Sprite extends TextureRegion {
 
 	final float[] vertices = new float[SPRITE_SIZE];
 	private final Color color = new Color(1, 1, 1, 1);
+	private float packedColor = Color.WHITE_FLOAT_BITS;
```

Comment:
In general storing the packed color seems OK. Do you see a performance difference? I vaguely remember looking into avoiding color/float conversions in the past. I don't remember the outcome, but maybe the performance difference was negligible.

Reply:
I haven't attempted to avoid color conversions yet to compare, in my case I'd probably need to make changes to `ParticleEmitter` as well.

I can't discard that it gets optimized by the compiler on a release build (haven't tested yet because I lack an Android Q device atm) and I don't have any data on iOS either (no idea how the RoboVM AOT compiler compares to the Android one in terms of performance optimizations). 

Testing performance properly on different scenarios and platforms is hard (and takes time!). Good thing in this case is the change doesn't come with any drawback.